edelphi/src/main/webapp/_scripts/gui/panel_admin/manageplanblockcontroller.js

```diff
@@ -0,0 +1,41 @@
+/*global CONTEXTPATH,BlockController,JSONUtils,addBlockController,startLoadingOperation,endLoadingOperation */
+(function() {
+  'use strict';
+  
+  var PlanEditorBlockController = Class.create(BlockController, {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') "BlockController" does not exist. Change its name or declare it so that its usage doesn't result in a "ReferenceError". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS3827)